src/resources/views/admin_add_user.blade.php

```diff
@@ -3,18 +3,113 @@
+                              @enderror
+                          </div>
+                      </div>
+                      <!-- Role (dropdown) -->
+                      <div class="form-group row">
+                          <label for="role" class="col-md-4 col-form-label text-md-right">{{ __('Role') }}</label>
+                          <div class="col-md-6">
+                              <select class="" name="role">
+                                  <option value="student">Student</option>
+                                  <option value="admin">Admin</option>
+                                  <option value="expert">Expert</option>
+                                  <option value="ta">TA</option>
+                              </select>
+                          </div>
+                      </div>
+
+                      <div class="form-group row mb-0">
+                          <div class="col-md-6 offset-md-4">
+                              <button type="submit" name="add_single_user" class="btn btn-primary">
+                                  {{ __('Add User') }}
+                              </button>
+                          </div>
+                      </div>
+                      <br>
+
+                      @if ($message = Session::get('add_message'))
+                       <div class="alert alert-success">
+                         <strong>{{ $message }}</strong>
+                       </div>
+                      @endif
+
+                  </form>
+                </div>
+            </div>
+        </div>
+
 
-                    {{ __('Add users if you are an admin!') }}
+        <div class="col-md-6">
+            <div class="card">
+                <div class="card-header">{{ __('Add Multiple Users') }}</div>
+                <div class="card-body" align="center">
+                    <div align="center">
+                        <form method="get" action="template_files/add_user_template.csv">
+                            <button type="submit" class="btn btn-primary">
+                                {{ __('Download Template File') }}
+                            </button>
+                        </form>
+                    </div>
+                    <br>
+                    <div align="center">
+                        <p style="text-align:center; color: black; background-color: #f7f7f7;border-radius: 5px;border: 1px solid #dfdfdf;">Add Multiple Users from a File</p>
+                        <form action="/add_user" method="post" enctype="multipart/form-data">
+                            @csrf
+                            <input type="file" name = "mycsv" id="users_upload"/>
+                            <input type="submit" class="btn btn-secondary" value="Upload">
+                        </form>
+                    </div>
+                    <br>
+
+                    @if ($message = Session::get('user_count_message'))
+                     <div class="alert alert-success">
```

Comment:
Maybe if all users were added successfully, we can show a success alert, if only some were added successfully we can show a caution and alert and if there was an error either parsing the file or all emails were taken, then we should display an error message.

Reply:
![image](https://user-images.githubusercontent.com/43078156/107840544-f303ed00-6d89-11eb-9560-8182dd309cc0.png)
![image](https://user-images.githubusercontent.com/43078156/107840549-fe571880-6d89-11eb-9835-e8cdaaf80206.png)
![image](https://user-images.githubusercontent.com/43078156/107840566-2b0b3000-6d8a-11eb-98ad-ccf4d5d06bbe.png)

Done!